news-bundle/tests/Controller/Page/NewsFeedControllerTest.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Controller\Page;
```

Comment:
This seems wrong. It should probably be `Contao\NewsBundle\Tests\ Controller\Page`.

Reply:
Fixed in a8cda0c9c